PathOfBuildingCommunity/PathOfBuilding
# ux: Add icons to Socket Group skill list based on the item slot

Fixes #5429

### Description of the problem being solved:

I was tired of clicking each items to know where they are equipped whenever i import a new build

Now it's properly displayed all at once, so i get a better understanding of where things are

LICENCE for icons: CC0 - https://ashizian.itch.io/2-bits-pack

### Steps taken to verify a working solution:
- Check Socket Group list in Skills tab
- Notice each items display proper icon based on the socket group's slot

### Link to a build that showcases this PR:

Any build will work

### Before screenshot:

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/44361234/6cf692e9-3eb1-4b49-ae2b-b2e65d2b4c44)


### After screenshot:

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/44361234/a1597516-bb4b-4f43-baf6-6e66108916f7)


![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/44361234/6deb1ce0-cf7a-4309-a66f-65563638821c)
